### DISCUSSION TITLE
Installation on WSL2 (Ubuntu)

### DISCUSSION THREAD

**Discussion by stritti** [remotemobprogramming/mob#390] [Show and tell]
Short step by step how to install on WSL2 (Windows Subsystem for LINUX) using Debian:

1. install speaking feature for timer:
   ```sh
    sudo apt-get install espeak-ng-espeak mbrola-us1
    ```
2. add `say` command:
   ```sh
   sudo nano /usr/bin/say
   ```
   and add content:
   ```sh
   #!/bin/sh
   espeak -v us-mbrola-1 "$@"
   ```
3. update permission to be executable:
   ```sh
   sudo chmod +x /usr/bin/say
   ```
4. install mob itselfe:
   ```sh
   curl -sL install.mob.sh | sudo sh
   ```

**Comment by hollesse** [comment]
I think we could add this to the README :)
